vcell-client/src/main/java/cbit/vcell/mapping/gui/LinkSpecsTableModel.java

```diff
@@ -260,6 +259,24 @@ protected List<MolecularInternalLinkSpec> computeData() {
         } else {
             return null;
         }
+
+        Collections.sort(allParameterList, new Comparator<MolecularInternalLinkSpec>() {
+            @Override
+            public int compare(MolecularInternalLinkSpec mils1, MolecularInternalLinkSpec mils2) {
+                SpeciesContextSpec scs = mils1.getSpeciesContextSpec();
+                if(fieldSpeciesContextSpec != scs) {
+                    throw new RuntimeException("SpeciesContextSpec inconsistent.");
+                }
```

Comment:
do you need to verify that mils2 also points to the correct SpeciesContextSpec?  it is okay if you think that is overkill.

Reply:
Actually even the test for mils1 is overkill, I'm only trying to catch bugs early during development. Condition should bever happen in a bug-free code.